schemas/network.yml

```diff
@@ -138,3 +138,20 @@
 
         If `source.packets` and `destination.packets` are known, `network.packets` is their sum.
       example: 24
+    
+    # q-in-q vlan fields for identifying 802.1q nested vlans
+    - name: inner
+      level: extended
+      type: object
+      short: Inner VLAN information
+      description: >
+        Network.inner fields are used to describe the innermost VLAN when q-in-q
+        multiple VLAN tagging is present.
+
+    - name: outer
+      level: extended
+      type: object
+      short: Outer VLAN information
+      description: >
+        Network.outer fields are used to describe the outermost VLAN when q-in-q
```

Comment:
Please add short descriptions to these two

Reply:
done